src/layers/BitmapLayer.js

```diff
@@ -0,0 +1,98 @@
+import { BitmapLayer as BaseBitmapLayer } from '@deck.gl/layers';
+import { COORDINATE_SYSTEM } from '@deck.gl/core';
+import { Model, Geometry } from '@luma.gl/core';
+import GL from '@luma.gl/constants';
+
+const PHOTOMETRIC_INTERPRETATIONS = {
+  WhiteIsZero: 0,
+  BlackIsZero: 1,
+  RGB: 2,
+  Palette: 3,
+  TransparencyMask: 4,
+  CMYK: 5,
+  YCbCr: 6,
+  CIELab: 8,
+  ICCLab: 9
+};
+
+const defaultProps = {
```

Comment:
Probably just want to spread the default props for `BaseBitmapLayer` and add a defaultProp for photometric interpretation of RGB?

```javascript
const defaultProps = {
  ...BaseBitmapLayer.defaultProps,
  pickable: true,
  coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
  bounds: { type: 'array', value: [0, 0, 1, 1], compare: true },
  photometricInterpretation: PHOTOMETRIC_INTERPRETATIONS.RGB,
};

```

Reply:
This means our layer _should_ be totally interchangeable with the regular `BitmapLayer`, but with the extra supported prop of `photometricInterpretation` that has a default of `RGB` rather than throwing an error.

Reply:
It inherits props automatically.  It's a mistake that I put that in the `MultiscaleImageLayer`